mixmatch/proxy.py

```diff
@@ -23,62 +25,55 @@
 from mixmatch import model
 from mixmatch import services
 
-GLANCE_APIS = ['images', 'schemas', 'metadefs', 'tasks']
-
 
 def stream_response(response):
     yield response.raw.read()
 
 
+def is_valid_uuid(value):
+    try:
+        _ = uuid.UUID(value, version=4)  # noqa
```

Comment:
Why not just ``uuid.UUID(value, version=4)`` as the whole line?  would that fix the pep8 issue?